### DISCUSSION TITLE
Cannot read properties of null (reading 'fingerprint') after calling Filament::notify()

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#1627] [Help]
Returns the error when calling ``` Filament::notify('success', 'Saved!'); ``` in the Livewire component.

Error:
```diff
- Uncaught (in promise) TypeError: Cannot read properties of null (reading 'fingerprint')
```

**Comment by studiowizjo** [comment]
Error points to "fingerprint", so check out value of this property. There are no details to say more, so please provide code of this component.
